test/mocha.opts

```diff
@@ -2,6 +2,7 @@
 --reporter spec
 --recursive
 --check-leaks
+--globals i
```

Comment:
It's in the commit message -- this is how you ignore a "namespace leak" failing list from the "node-escapist" module.  I see that module has been removed from Github, so the link in the commit message is broken.  I think we'll stop using this when we move the UA service to CLJS.